java/pom.xml

```diff
@@ -24,7 +24,7 @@ SPDX-License-Identifier: Apache-2.0
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <junit.version>5.4.2</junit.version>
-    <jcuda.version>12.0.0</jcuda.version>
+    <jcuda.version>12.2.2</jcuda.version>
```

Comment:
Cannot tell if this or other Java files will need changes. Would be great to hear feedback from Java developers

Reply:
Looks like [`jcuda` on Maven]( https://repo.maven.apache.org/maven2/org/jcuda/jcuda/ ) can only be 12.0.0 or 12.6.0. Please let us know if you have thoughts on that and any other Java changes that would be needed